### PR TITLE
tags/instructor/*.tag: change to 2-space indentation

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes partially #7352 
All leftover files in tags/instructor/*.tag

### CODE REVIEW

**Comment by sujeet14108** [comment]
Ready for review :)

File: src/main/webapp/WEB-INF/tags/instructor/copyModal.tag [hunk 1]
```diff
@@ -3,27 +3,27 @@
...
+          </h4>
         </div>
+        <div class="modal-body">
+          <div id="courseList" class="form-group"></div>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+          <input type="submit" class="btn btn-primary" id="fscopy_submit" value="Copy">
+          <input type="hidden" name="<%= Const.ParamsNames.USER_ID %>" value="${data.account.googleId}">
+         </div>
```
Comment (chowyb) [inline src/main/webapp/WEB-INF/tags/instructor/copyModal.tag]: Remove 1 space from this line.

File: src/main/webapp/WEB-INF/tags/instructor/navBar.tag [hunk 2]
```diff
@@ -4,48 +4,48 @@
...
+          <a class='nav search' data-link="instructorSearch" href="${data.instructorSearchLink}">
+            Search
+          </a>
+        </li>
+        <li>
+          <a class="nav help" href="/instructorHelp.jsp" target="_blank" rel="noopener noreferrer">Help</a>
+        </li>
+      </ul>
+      <ul class="nav navbar-nav pull-right">
+        <li><a id="btnLogout" class="nav logout" href="<%= Const.ActionURIs.LOGOUT %>">Logout
```
Comment (chowyb) [inline src/main/webapp/WEB-INF/tags/instructor/navBar.tag]: I'd suggest shifting the opening `<a>` tag down to a new line here, even though it was on the same line in the old code.

File: src/main/webapp/WEB-INF/tags/instructor/studentList.tag [hunk 3]
```diff
@@ -8,188 +8,188 @@
...
+              </c:if>
+              <tr class="student_row" id="student-c${courseIndex}.${studentIndex}">
+                <td id="studentphoto-c${courseIndex}.${studentIndex}">
+                  <div class="profile-pic-icon-click align-center" data-link="${student.photoUrl}">
+                    <a class="student-profile-pic-view-link btn-link">View Photo</a>
+                    <img src="" alt="No Image Given" class="hidden">
+                  </div>
+                </td>
+                <td id="studentsection-c${courseIndex}.${sectionIndex}"
+                  <c:if test="${not hasSection}">class="hidden"</c:if>>
```
Comment (chowyb) [inline src/main/webapp/WEB-INF/tags/instructor/studentList.tag]: Add an extra 2 spaces of indentation to this line.

File: src/main/webapp/WEB-INF/tags/instructor/studentList.tag [hunk 4]
```diff
@@ -8,188 +8,188 @@
...
+                            title="<%= Const.Tooltips.ACTION_NOT_ALLOWED %>"
+                            disabled
+                          </c:when>
+                          <c:otherwise>
+                            title="<%= Const.Tooltips.COURSE_STUDENT_REMIND %>"
+                            href="${student.courseStudentRemindLink}"
+                          </c:otherwise>
+                        </c:choose>
+                       data-toggle="tooltip"
+                       data-placement="top">
```
Comment (chowyb) [inline src/main/webapp/WEB-INF/tags/instructor/studentList.tag]: Add an extra space of indentation to these 2 lines.

File: src/main/webapp/WEB-INF/tags/instructor/studentList.tag [hunk 5]
```diff
@@ -8,188 +8,188 @@
...
+                      Send Invite
+                    </a>
+                  </c:if>
+                  <c:set var="deleteButtonEnabled" value="${section.allowedToModifyStudent}" />
+                  <a class="course-student-delete-link btn btn-default btn-xs margin-bottom-7px<c:if test="${not deleteButtonEnabled}"> disabled mouse-hover-only</c:if>"
+                    data-student-name="${student.studentNameForJs}" data-course-id="${student.courseIdForJs}"
```
Comment (chowyb) [inline src/main/webapp/WEB-INF/tags/instructor/studentList.tag]: Add 2 spaces of indentation to this line.

**Comment by sujeet14108** [comment]
@chowyb Please review :)

File: src/main/webapp/WEB-INF/tags/instructor/home/sort.tag [hunk 6]
```diff
@@ -6,33 +6,33 @@
...
-                       name="sortby" data="createdAt" id="sortByDate"
-                       <c:if test="${isSortButtonsDisabled}">disabled</c:if>>
-                    <input type="radio">
-                    Creation Date
-                </label>
-            </div>
-        </div>
+  <div class="col-xs-2">
+    <a class="btn btn-primary btn-md" href="${data.instructorCoursesLink}"
+      id="addNewCourse">Add New Course</a>
```
Comment (chowyb) [inline src/main/webapp/WEB-INF/tags/instructor/home/sort.tag]: Add 2 spaces to this line.

File: src/main/webapp/WEB-INF/tags/instructor/moreInfo.tag [hunk 7]
```diff
@@ -5,33 +5,33 @@
...
-                <h5>More Info </h5>
-                <%-- Note: When an element has class text-preserve-space, do not insert HTML spaces --%>
-                <p class="text-preserve-space height-fixed-md">${empty student.moreInfo ? notFilled : fn:escapeXml(student.moreInfo)}</p>
-            </div>
-        </div>
+  <div class="col-xs-12">
+    <div class="panel panel-default">
+      <div class="panel-body">
+        <span data-toggle="modal" data-target="#studentProfileMoreInfo"
+          class="text-muted pull-right glyphicon glyphicon-resize-full cursor-pointer"></span>
```
Comment (chowyb) [inline src/main/webapp/WEB-INF/tags/instructor/moreInfo.tag]: Add 2 spaces to this line.

Comment (sujeet14108) [inline src/main/webapp/WEB-INF/tags/instructor/moreInfo.tag]: Done

**Comment by sujeet14108** [comment]
@chowyb Ready for review

**Pull request merged**